source/includes/usage-examples/read-sample-app.py

```diff
@@ -0,0 +1,18 @@
+from pymongo import MongoClient
+
+uri = "<connection string URI>"
+client = MongoClient(uri)
+
+try:
+    database = client["<database name>"]
+    collection = database["<collection name>"]
+
+    # start example code below
```

Comment:
S: Might be good to be consistent with our comments here. I'm flexible on what they say, but the auth examples say "start authentication code here" instead of "below"

Reply:
Oops, yeah I meant to change that to match the auth ones